src/v/raft/prevote_stm.cc

```diff
@@ -119,7 +118,7 @@ ss::future<bool> prevote_stm::prevote(bool leadership_transfer) {
       .with([this, leadership_transfer] {
           _config = _ptr->config();
           _config->for_each_voter(
-            [this](model::node_id id) { _replies.emplace(id, vmeta{}); });
+            [this](vnode id) { _replies.emplace(id, vmeta{}); });
```

Comment:
I think we should `emplace() + shrink_to_fit()` it's more expensive, but honestly, it's mostly a lookup datastructure. after a stable cluster, it's just for lookups.